src/index.js

```diff
@@ -132,6 +139,21 @@ export default class Firedux {
         }))
       }
 
+      if (this.v3) {
+        const auth = this.auth()
+
+        auth.onAuthStateChanged(function (user) {
+          if (user) {
+            resolve(user)
```

Comment:
We should probably persist the `user` here either as `this.authData` or some new property.